icu4j/readme.html

```diff
@@ -21,17 +21,17 @@ <h2>Read Me for ICU4J 69.1</h2>
 <p>
 <b>Note:</b> This is a major release of ICU4J. It contains bug fixes and adds implementations
 of inherited API and introduces new API or functionality.
-<!-- <b>Note:</b> This is a preview release of ICU4J 69.
+<b>Note:</b> This is a preview release of ICU4J 69.
 The contents of this document may not reflect the recent changes done
-for ICU 69 development. It is not recommended for production use. -->
-<!-- <b>Note:</b> This is a development milestone of ICU4J 69.
```

Comment:
Ah, we only want to uncomment the note about the release candidate. Otherwise we'll end up with *all* the notes on the readme page showing. :)